app/views/runs/graphs/index.slim

```diff
@@ -0,0 +1,33 @@
+- title(@run.name)
+= render partial: 'runs/twitter_card'
+
+- content_for(:header) do
+  h3
+    - if @run.game.try(:name).present? && @run.category.try(:name).present?
+      = link_to @run.game, game_path(@run.game), class: 'stealth-link game-link'
+      =< link_to @run.category, game_category_path(@run.game, @run.category), class: 'stealth-link category-link'
+    - else
+      = @run.name
+  h5
+    .time.needs-formatting class=('short' if @run.short?) = @run.time
+    - if @run.user.try(:name)
+      small by #{link_to @run.user, user_path(@run.user), class: 'stealth-link user-link'}
+  h6
+    ul.run-options
+      li
+        a href=run_path(@run) &larr; back to run
+
+- if @run.time > 0
+  #timeline data-turbolinks-permanent=true
+    = render partial: 'runs/timeline', locals: {run: @run}
+    - if @run.splits.all? { |split| !split.skipped? } && @run.has_golds?
+      = render partial: 'runs/gold_timeline', locals: {run: @run}
+  #timeline-inspector data-turbolinks-permanent=true
+    = render partial: 'runs/timeline_inspector', locals: {run: @run}
+
+article#graphs
+  #pb-graph
+  #split-history-graph
+  #run-duration-graph
+  #segment-duration-graph
+  #reset-graph
```

Comment:
can we put some `hr`s or spacing between each graph? as it stands it's hard to tell at a glance if the titles belong to the graph above them or the one below them.

<img width="1270" alt="screenshot 2016-08-09 09 45 12" src="https://cloud.githubusercontent.com/assets/438911/17525061/ff96fd06-5e15-11e6-9cf7-66acc60fee0e.png">